app/src/pages/outside/loginPage/loginPage.jsx

```diff
@@ -88,6 +100,9 @@ export class LoginPage extends PureComponent {
     if (multipleAuth) {
       currentBlock = <MultipleAuthBlock multipleAuthKey={multipleAuth} />;
     }
+    if (registration && extensions) {
+      currentBlock = extensions.map((extension) => <ExtensionLoader extension={extension} />);
```

Comment:
Don't forget to provide `key` to the each component of collection.

Reply:
Please note that `ServiceVersionsBlock` should not be rendered only together with  `LoginBlock`.